common/traits/btr_species_traits.txt

```diff
@@ -820,15 +828,68 @@ btr_yahg_keen_mind = {
 	}
 
 	modifier = {
-		species_leader_exp_gain = 0.25
-		planet_jobs_engineering_research_produces_mult = 0.2
-		planet_jobs_physics_research_produces_mult = 0.2
-		planet_jobs_society_research_produces_mult = 0.2
-		intel_decryption_mult = 0.2
-		intel_encryption_mult = 0.2
+		pop_growth_speed = -0.15
+		planet_pops_organics_food_upkeep_mult = 0.15
+		pop_housing_usage_mult = 0.25
+		army_damage_mult = 0.66
+		army_morale = 0.66
 	}
 }
 
+
+btr_yahg_trait_truthsay = {
+	cost = 0
+
+	#Opposites TBD
+	opposites = {
+		"trait_intelligent"
+		"trait_natural_physicists"
+		"trait_natural_engineers"
+		"trait_natural_sociologists"
+	}
+
+	initial = yes
+	randomized = no
+	modification = no
+	allowed_archetypes = {
+		BIOLOGICAL
+	}
+
+	modifier = {
+		planet_jobs_society_research_produces_mult = 0.15 
+		empire_size_mult = -0.20 
```

Comment:
```suggestion
		empire_size_pops_mult= -0.20 
```

The correct modifier is probably empire_size_pops_mult.

Reply:
Yea I think your probably right